src/components/contents/ListView.jsx

```diff
@@ -17,13 +24,9 @@ function ListView({ user, currentDBId, setDocumentsIds }) {
     ["dbDocumentList", currentDBId],
     getDocumentsList,
     {
-      enabled: !!user,
+      enabled: !!userId,
       onSuccess: result => {
-        const newArr = [];
-
-        result.forEach(element => {
-          newArr.push(element._id);
-        });
+        const newArr = result.map(element => element._id);
```

Comment:
로직을 한 단계 더 효율적으로 작성하셨네요 🥳

`newArr` 네이밍을 좀더 구체적인 이름으로 수정해보는건 어떨까요? 🤔

document들을 `result` 배열로 받아와서 각각의 element의 _id 값을 `newArr` 배열에 담아주고 있는 로직으로 보이는데, 그러면 `documentIds`는 어떨지 제안드려봅니다.

```suggestion
        const documentIds = result.map(document => document._id);
```

Reply:
shallow copy하고 수정한후 replace해주는 로직에서 newArr 를 써주고 있었는데요
이 부분은 그에 해당하지 않으니 제안하신 대로 documentIds로 바꾸는게 더 정확하겠네요. 
지적 정말 감사드립니다~!